docs/maintenance/Pull_Requests.mdx

```diff
@@ -24,15 +24,85 @@ Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#poin
 - "Unusual"-categorized: require two maintainer approvals.
   These include significant refactors with cross-project and public API ramifications.
 
-## PR Flow
+## PR Review Flow
 
 :::note
 We include a set of common responses to PRs in [`.github/replies.yml`](https://github.com/typescript-eslint/typescript-eslint/blob/main/.github/replies.yml), intended to be used with the [Refined Saved Replies](https://github.com/JoshuaKGoldberg/refined-saved-replies) extension.
 Don't treat these as exact responses you must use: they're just a starting copy+paste helper.
 Please do adopt your specific responses to your personal tone and to match the thread for non-straightforward PRs.
 :::
 
-TODO: This will be filled out... soon!
+Open pull requests generally switch between two states:
+
+- Ready for review: either newly created or after [review is re-requested](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review)
+- Waiting for author activity: either by virtue of [being a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) or having the [`awaiting response` label](https://github.com/typescript-eslint/typescript-eslint/pulls?q=is%3Apr+is%3Aopen+label%3A%22awaiting+response%22)
+
+Add the `awaiting response` label to a PR whenever you post a review.
+It will be automatically removed if the author re-requests review.
+
+Authors sometimes separately ask questions as comments on PRs, sometimes including an `@` tag.
+Put back the `awaiting response` label if you answer the questions.
+Don't worry if you miss some of these questions; we prefer going through the formal re-requesting review to make sure the `awaiting response` label is removed as needed.
+
+### Be Kind
+
+Above all else, be _encouraging_ and _friendly_ in tone.
+
+- Phrase feedback as opportunities for improvement rather than chastising.
+- Call out any positive aspects you see - especially in larger pull requests.
+- Use happy emojis frequently.
+- If you have the energy, post a fun (but not inappropriate) GIF with your review.
+  - We often use the _GIFs for Github_ extension: available as [GIFs for GitHub (Chrome)](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep) and [GIFs for GitHub (Firefox)](https://addons.mozilla.org/en-US/firefox/addon/gifs-for-github).
+
+Pull requests are the first time many community members meaningfully interact with us - or, oftentimes, any open source maintainers at all.
+It's important that we provide a positive, uplifting experience. ❤️
+
+### Reviewing a PR
+
+Before reviewing a pull request, try to look back over the backing issue to (re-)familiarize yourself with it.
```

Comment:
Ah, I think it would be great to cover cases where an issue doesn't exist or isn't marked as `accepting prs` (and the changes made in the PR aren't just a `chore: fix typo`), like here https://github.com/typescript-eslint/typescript-eslint/pull/8322#issuecomment-1925462761. What do you think?

Reply:
Love it!